nginx/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM clearlinux:latest
+MAINTAINER bin.yang@intel.com
+
+RUN swupd update $swupd_args && \
+    swupd bundle-add nginx $swupd_args
+RUN rm -rf /var/lib/swupd
+
+# forward request and error logs to docker log collector
+RUN mkdir -p /var/log/nginx
+RUN ln -sf /dev/stdout /var/log/nginx/access.log \
+	&& ln -sf /dev/stderr /var/log/nginx/error.log
+
+RUN mkdir -p /var/lib/nginx/
+RUN mkdir -p /etc/nginx/conf.d
+RUN cp -f /usr/share/nginx/conf/nginx.conf.example /etc/nginx/nginx.conf
+RUN echo 'server {' > /etc/nginx/conf.d/default.conf
+RUN echo '  server_name localhost;' >> /etc/nginx/conf.d/default.conf
+RUN echo '  location / {' >> /etc/nginx/conf.d/default.conf
+RUN echo '    root /usr/share/nginx/html;' >> /etc/nginx/conf.d/default.conf
+RUN echo '    autoindex on;' >> /etc/nginx/conf.d/default.conf
+RUN echo '  }' >> /etc/nginx/conf.d/default.conf
+RUN echo '}' >> /etc/nginx/conf.d/default.conf
```

Comment:
Is there a reason why we aren't just COPYing this file in? All these RUN commands create a new layer.

Reply:
yes, it is better to use COPY here. I will update PR